client/src/components/StreamField/blocks/FieldBlock.js

```diff
@@ -139,13 +158,22 @@ export class FieldBlockDefinition {
     this.meta = meta;
   }
 
-  render(placeholder, prefix, initialState, initialError) {
+  render(
+    placeholder,
+    prefix,
+    initialState,
+    initialError,
+    parent,
+    parentCapabilities,
+  ) {
     return new FieldBlock(
       this,
       placeholder,
       prefix,
       initialState,
       initialError,
+      parent,
```

Comment:
Looks like there's some inconsistency here about whether `render` and `FieldBlock` accept a `parent` argument in addition to `parentCapabilities` - my take is that they shouldn't (because that would be giving the child block too much knowledge about the parent), in which case this method is wrong, and the call to `render` from StructBlock is incorrect, and everything else is correct.

(The effects of this inconsistency mostly cancel out, because ListBlock/StreamBlock end up passing the capabilities object as the 6th argument, which is what FieldBlock wants; and StructBlock's capabilities object has no usable properties, which means passing `this` in its place is functionally identical...)

Reply:
Thanks for catching this! You're right, this was a remnant of the event listener approach I'd missed, and I agree they shouldn't be getting parent knowledge like that - should all be removed now.